tests/Rules/DisallowedConstructs/DisallowedWeakComparisonRuleTest.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\DisallowedConstructs;
+
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+class DisallowedWeakComparisonRuleTest extends RuleTestCase
```

Comment:
I think you can get rid of the new phpstan errors added to the baseline if you specify the  generic type in the same fashion as it is done in e.g. https://github.com/phpstan/phpstan-strict-rules/blob/1.2.3/tests/Rules/DisallowedConstructs/DisallowedShortTernaryRuleTest.php#L9

Reply:
OK, I don't mind which variant I use. I just took an example from the existing test classes.